app/helpers/menu_helper.rb

```diff
@@ -127,15 +127,17 @@ def masthead_props
 
   def context_selector_props
     title, icon = active_title_and_icon
+    access_to_service_admin_sections = current_user.access_to_service_admin_sections?
```

Comment:
The test for this is failing, I think it's only a matter of stubbing `current-user`.